style/elixir/README.md

```diff
@@ -0,0 +1,11 @@
+Elixir
+======
+
+[Sample](sample.ex)
+
+* Avoid macros.
+* Don't separate multiple declarations of the same function with a newline.
```

Comment:
I'm not sure to be honest. It's something I've done and I threw it in here to get some feedback on it.